docker-compose.yml

```diff
@@ -16,5 +16,5 @@ services:
     env_file: docker.env
     environment:
       - SPEC_OPTS
-      #- JRUBY_OPTS
```

Comment:
You can clean up docker.env `JRUBY_OPTS`

Reply:
what do you mean, cleaning up comments at https://github.com/logstash-plugins/.ci/blob/master/docker.env ?
believe we want those around ...

Reply:
I mean removing docker.env line 6 `JRUBY_OPTS=-Xregexp...` because none of the file, docker-compose.yml nor Dockerfile, use `JRUBY_OPTS` anymore. The clean up is optional

Reply:
oh I see what you mean now. 
I've only removed the (commented out) `JRUBY_OPTS` here which, if set, would allow changing it for docker's environment from the host env.
the *docker.env* file is still read by docker-compose - here's a sample that those `JRUBY_OPTS` [are being used](https://travis-ci.org/github/logstash-plugins/logstash-output-http/jobs/740309252#L455).